tests/lib/northbound/test_oper_data.c

```diff
@@ -400,7 +400,7 @@ int main(int argc, char **argv)
 	vty_init(master, false);
 	lib_cmd_init();
 	yang_init(true);
-	nb_init(master, modules, array_size(modules));
+	nb_init(master, modules, array_size(modules), true);
```

Comment:
nit: please use `false` here as well, since we don't want unit tests to record transactions under any circumstance. It should make any difference in this particular case but it's better to keep things consistent.